internal/cmd/all/list.go

```diff
@@ -0,0 +1,157 @@
+package all
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/hetznercloud/cli/internal/cmd/base"
+	"github.com/hetznercloud/cli/internal/cmd/certificate"
+	"github.com/hetznercloud/cli/internal/cmd/firewall"
+	"github.com/hetznercloud/cli/internal/cmd/floatingip"
+	"github.com/hetznercloud/cli/internal/cmd/image"
+	"github.com/hetznercloud/cli/internal/cmd/iso"
+	"github.com/hetznercloud/cli/internal/cmd/loadbalancer"
+	"github.com/hetznercloud/cli/internal/cmd/network"
+	"github.com/hetznercloud/cli/internal/cmd/output"
+	"github.com/hetznercloud/cli/internal/cmd/placementgroup"
+	"github.com/hetznercloud/cli/internal/cmd/primaryip"
+	"github.com/hetznercloud/cli/internal/cmd/server"
+	"github.com/hetznercloud/cli/internal/cmd/sshkey"
+	"github.com/hetznercloud/cli/internal/cmd/volume"
+	"github.com/hetznercloud/cli/internal/hcapi2"
+	"github.com/hetznercloud/cli/internal/state"
+	"github.com/hetznercloud/hcloud-go/v2/hcloud"
+)
+
+var listCmd = base.Cmd{
+	BaseCobraCommand: func(client hcapi2.Client) *cobra.Command {
+		cmd := &cobra.Command{
+			Use:   "list FLAGS",
+			Short: "List all resources",
+		}
+
+		cmd.Flags().StringP("selector", "l", "", "Selector to filter by labels")
+
+		cmd.Flags().Bool("paid", false, "Only list paid resources")
```

Comment:
If I may shim in here: what does `paid` refer to ? Is that to exclude resources that were not created by the user ? e.g. official images/isos ?